lib/guests.js

```diff
@@ -8,40 +8,28 @@ function guests(bot, message) {
   debug('begin');
   let user = message.user;
   let getData = Promise.promisify(bot.botkit.storage.users.get);
+  let guestsInviteResults = {
+    ok: 'Invitado',
+    already_invited: 'Ya se invitó',
+    already_in_team: 'Ya está registrado',
+  };
 
   return getData(user)
     .then((data) => {
       debug('pana', data);
       debug('panas', data.guests);
 
       if (data.guests.length === 0) {
-        return bot.reply(message, 'Oiga, invitá un parcero primero!');
-      }
-
-      const activeGuests = data.guests.filter((guest) => guest.result === 'ok');
-      if (activeGuests.length === 0) {
-        return bot.reply(message, 'No hay parceros activos, invitá uno ome!');
+        return bot.reply(message, 'Oe, invitá un parcero primero!');
       }
 
       return bot.reply(message, [
           'Tus parceros:',
           '\n',
-          `${activeGuests.map((guest) => guest.guest).join('\n')}`,
+          `${data.guests.map((guest) => `${guest.guest}: ${guestsInviteResults[guest.result]}`).join('\n')}`,
```

Comment:
how about just show the message stored in the result, so users can see?

On Tue, Sep 6, 2016 at 12:32 PM Fernando Montoya <notifications@github.com>
wrote:

> In lib/guests.js
> <https://github.com/colombia-dev/leonel/pull/41#discussion_r77670521>:
>
> >        }
> >
> >        return bot.reply(message, [
> >            'Tus parceros:',
> >            '\n',
> > -          `${activeGuests.map((guest) => guest.guest).join('\n')}`,
> > +          `${data.guests.map((guest) => `${guest.guest}: ${guestsInviteResults[guest.result]}`).join('\n')}`,
>
> It could show a empty space. What you think?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/colombia-dev/leonel/pull/41/files/1247cdb6a0a3848a23a84bbc3f4e7b0dafb5c1d3..d916980a30d01d9d0b7291eb8808c753d3928ea2#r77670521>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAN7GCYQlZid2JglVDK2QooCECKwyjQKks5qnZWBgaJpZM4J0KDk>
> .
>